src/StageConfig.h

```diff
@@ -0,0 +1,99 @@
+#ifndef MBMORE_SRC_STAGE_H_
+#define MBMORE_SRC_STAGE_H_
+
+#include <string>
+#include "CentrifugeConfig.h"
+
+namespace mbmore {
+class CascadeConfig;
+
+class StageConfig {
+
+  private:
+
+    // Compute the cut to ensure alpha = beta (from dU)
+    void CutForIdealStg();
+
+    // Precision used for the cut calculation defautl 1e-8
+    double precision_ = 1e-8;
+
+    // cut value of the stage
+    double cut_;
+    // dU value of the stage (calculated form the centrifuges config with the cut)
+    double DU_;
+    // Feed to Product enrichment ratio
+    double alpha_;
+    // Feed to Tail enrichment ratio
+    double beta_;
+    // Feed flow (g/s)
+    double feed_flow_;
+
+    // number of centrifuges in the stage
+    int n_machines_;
```

Comment:
```suggestion
    int n_machines_ = -1;
```

Set default to clearly define variable before `MachinesNeededPerStage(...)` has been called